.github/workflows/release-build.yml

```diff
@@ -31,3 +31,34 @@ jobs:
 
       - name: Publish
         run: make VERSION=${GITHUB_REF#refs/tags/v} publish
+
+        # Get release information to determine id of the current release
+      - name: Get Release
+        id: get-release-info
+        uses: bruceadams/get-release@v1.2.0
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+
+      # Upload deployment YAML files as ZIP to GitHub release
+      - name: Upload Deployment YAML (zip)
+        id: upload-deployment-yaml-zip
+        uses: actions/upload-release-asset@v1
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        with:
+          upload_url: https://uploads.github.com/repos/tomkerkhove/keda-sandbox/releases/${{ steps.get-release-info.outputs.id }}/assets?name=keda-1.4.1.zip
+          asset_path: keda-1.4.1.zip
+          asset_name: keda-1.4.1.zip
+          asset_content_type: application/zip
+
+      # Upload deployment YAML files as TAR to GitHub release
+      - name: Upload Deployment YAML (zip)
+        id: upload-deployment-yaml-tar
+        uses: actions/upload-release-asset@v1
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        with:
+          upload_url: https://uploads.github.com/repos/tomkerkhove/keda-sandbox/releases/${{ steps.get-release-info.outputs.id }}/assets?name=keda-1.4.1.tar.gz
+          asset_path: keda-1.4.1.tar.gz
```

Comment:
@zroubalik I'll need help with this, where does MAKE output these?

Reply:
to the root of the project eg `./keda-1.4.1.tar.gz`

Reply:
Then we should be good to go, was same on https://github.com/tomkerkhove/keda-sandbox